assets/js/googlesitekit/modules/datastore/__fixtures__/index.js

```diff
@@ -32,11 +32,28 @@ const alwaysActive = [ 'search-console', 'site-verification' ];
  * @param {...string} slugs Active module slugs.
  * @return {Object[]} Array of module objects.
  */
-export const withActive = ( ...slugs ) => {
+export function withActive( ...slugs ) {
 	const activeSlugs = alwaysActive.concat( slugs );
```

Comment:
Just a note: After (or with) #2184 we will be able to remove the `alwaysActive` variable here and instead rely on the `forceActive` property in the fixture to know which modules should be always active.

cc @asvinb Could you consider that in your PR for #2184 please?